src/components/organisms/AssetContent/MetaItem.module.css

```diff
@@ -10,3 +10,9 @@
   color: var(--color-secondary);
   text-transform: uppercase;
 }
+
+.content {
+  width: 11.5rem;
```

Comment:
double this is wider than most mobile viewports, resulting in scrollbar and generally misaligned things on whole screen:

<img width="567" alt="Screen Shot 2021-03-05 at 11 28 40" src="https://user-images.githubusercontent.com/90316/110103189-f2b5ab00-7da5-11eb-8b8d-13a139101516.png">

Setting fixed widths by default with our mobile first styles can become problematic later on. In this case I guess easiest is to put this width behind a `min-width` media query

Reply:
for reference: `grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));` on `.metaFull` and removing that `width` seems to solve all our problems